### ISSUE TITLE
AL-GO stopped respecting param Country in from .AL-Go/settings.json

### ISSUE THREAD

**Issue by gntpet** [microsoft/AL-Go#286]
Hey @freddydk, 

For some reason, AL-GO stopped respecting parameter Country within CI/CD pipeline. In my multi root project, i have project SRS.it, which has the following settings file:

```
{
    "country":  "it",
    "appFolders":  [
    ],
    "testFolders":  [
    ],
    "appDependencyProbingPaths":  [
        {
            "repo":  ".",
            "release_status":  "include",
            "projects":  "SRS.core"
        }
    ]
}
```

It used to build using italian artifacts, but for some reason it's using base version now. See screenshot from RunAlPipeline. It gives a strange warning, and for some reason event **at** artifact url

![image](https://user-images.githubusercontent.com/16342128/201772968-3dbc2f46-9eb2-4e88-bd25-85581acf8ed4.png)

See screenshot from Create Container. you can cleatly see it's using **base** version for creating container
![image](https://user-images.githubusercontent.com/16342128/201773421-4e92b5a0-8760-4ddc-8e86-bcc26385f184.png)
 

Please let me know if you need more info

**Comment by freddydk** [comment]
Please share your .github/AL-Go-Settings.json as well, thanks

**Comment by gntpet** [comment]
here you go:

```
{
    "type":  "PTE",
    "templateUrl":  "https://github.com/microsoft/AL-Go-PTE@preview",
    "artifact":  "bcartifacts/sandbox/21/base/latest",
    "treatTestFailuresAsWarnings":  true,
    "doNotRunBcptTests":  true
}
```

**Comment by freddydk** [comment]
"artifact":  "bcartifacts/sandbox/21/base/latest",

overrides the country.
replace base with it (and move it to .AL-Go settings)

But why did you add this?

**Comment by gntpet** [comment]
I added it, since I wanted to control which version I'm compiling against to. (artifacts for 21 are released before, our sandbox are updated to 21).

I guess, I did not notice that it started compiling IT version with base artifacts then. There was no real dependencies on IT version back then. Now I got those dependencies, and it lights up. 

To conclude, I should copy _artifact_ settings from  .github/AL-Go-Settings.json to each project. Lesson learned. 

Thank you for your help, @freddydk. I'm closing the ticket. False alarm
